packages/features/tasker/tasks/scanWorkflowBody.ts

```diff
@@ -0,0 +1,53 @@
+import { AkismetClient } from "akismet-api";
+import type { Comment } from "akismet-api";
+import z from "zod";
+
+import { lockUser } from "@calcom/lib/autoLock";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import logger from "@calcom/lib/logger";
+import prisma from "@calcom/prisma";
+
+export const scanWorkflowBodySchema = z.object({
+  userId: z.number(),
```

Comment:
Need the userId to know which user to lock. This comes from the context of the workflow update tRPC endpoint.